.expeditor/config.yml

```diff
@@ -62,17 +61,13 @@ subscriptions:
       - built_in:promote_habitat_packages
 
   # Subscribe to the promotion of the dev channel to acceptance
-  - workload: project_promoted:{{agent_id}}:dev:*
-    actions:
-      - built_in:promote_habitat_packages
-  - workload: project_promoted:{{agent_id}}:workstation-build:*
+  - workload: project_promoted:{{agent_id}}:unstable:*
     actions:
       - built_in:promote_habitat_packages
 
 pipelines:
   - verify:
       description: Pull Request validation tests
-      public: true
```

Comment:
This will remove it from the `chef-oss` Buildkite tenant and make it only visible to internal people